app/assets/stylesheets/_empty-list.scss

```
@@ -0,0 +1,5 @@
+.app-empty-list {
+  margin-top: nhsuk-spacing(6);
+  border: 2px solid #b1b4b6;
```

Does this look ok? Would be more consistent with the NHSUK Design System.

```suggestion
  border: 2px solid $nhsuk-border-color;
```

---

🤷 but I've changed to the design system default until we have a good reason to deviate...